lucene/version.properties

```diff
@@ -0,0 +1,10 @@
+# This file contains some version properties as used by various build files.
```

Comment:
Ülease remove that file, it wont be used.

If release scripts need to edit the version number after a release, <root>/build.gradle is your friend. Theres also a version.base and that's the source of information.

https://github.com/apache/lucene-solr/blob/master/build.gradle#L38

There is a reason why this file was removed, so dont restore it!